### ISSUE TITLE
Add document name to document.title

### ISSUE THREAD

**Issue by wesgarland** [benweet/stackedit#1522]
I use stackedit.io and frequently have several documents open in different tabs simultaneously.

It would be very useful to me if the tab title was either the filename or the title of the document or something like that, to make it easy to find what I'm looking for.

```javascript
document.title = 'StackEdit | ' + document.querySelector('.token.h1').innerText.slice(2)
```

looks pretty good on my box.

**Comment by bhagerty** [comment]
I agree that this would be a very nice feature, and it seems like it wouldn't bee that hard to implement. I find it challenging to navigate between Stackedit windows when the title bar of every window just says "Stackedit."

**Comment by mohamedahmedsurur** [comment]
![behance_project_1584006681488](https://user-images.githubusercontent.com/61625528/76798160-1d4ad480-67e0-11ea-9730-aab4e1a08baf.jpg)

**Comment by mohamedahmedsurur** [comment]
Mohamed Ahmed Surur
